### PR TITLE
SPLAT-1859: load vsphere ccm image from payload image for external provider tests.

### PR DESCRIPTION
**Pull request by vr4manta**
[SPLAT-1859](https://issues.redhat.com//browse/SPLAT-1859)

### Changes
- Modified vsphere platform external script to load the CCM image from payload image instead of using static defined image

### CODE REVIEW

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Comment by vr4manta** [comment]
Note: pj-rehearse failed for unit tests.  The changes for this PR completed install.  That is the primary fix we are looking for.

File: ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh [hunk 1]
```diff
@@ -94,6 +94,13 @@ export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
 # Ensure ignition assets are configured with the correct invoker to track CI jobs.
 export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}
 
+echo "$(date -u --rfc-3339=seconds) - Discovering controller image 'vsphere-cloud-controller-manager' from release [${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE-}]"
+
+PULL_SECRET="/var/run/secrets/ci.openshift.io/cluster-profile/pull-secret"
```
Comment (mtulio) [inline ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh:99]: nit: alternatively you can use the global env var: `"${CLUSTER_PROFILE_DIR}"/pull-secret`

Comment (vr4manta) [reply ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh:99]: oh, let me try that.  thanks!

**Comment by vr4manta** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.18-e2e-external-vsphere-ccm

**Review by mtulio — COMMENTED** [review]
/lgtm

**Pull request merged**